### DISCUSSION TITLE
Cyberduck is not displaying all the files available in Amazon S3

### DISCUSSION THREAD

**Discussion by MitchAnce** [iterate-ch/cyberduck#16186] [Q&A]
I am using Cyberduck to bulk download files from Amazon S3. The S3 is managed by an app called Customily which stores custom designs in the S3. 

However, Cyberduck is only displaying a small number of files compared to the number of files stored in S3. 

I tried with removing the connection to S3, re-adding the connection, uninstalling and re-installing Cyberduck. However, any of these fixed the issue. 

Has anyone have some experience with this kind of issue?

**Comment by ghost** [comment]
[cyberduck.log](https://github.com/user-attachments/files/16517045/cyberduck.log)

Cyberduck debug log above. Any suggestions on how to fix this would be greatly appreciated 🙏

**Reply by tatze96** [reply]
The attached log is an "Error" log, not a debug log. Also, it doesn't contain the faulty listing entries. Still the logging output does indicate access issues to the _Transfers_ folder within the [application support folder](https://docs.cyberduck.io/cyberduck/support/#application-support-folder). Please either move the folder out of the application support folder or delete it to see if this improves the behavior. Ensure that Cyberduck isn't running while removing the folder.

If this doesn't improve the behavior, activate [debug logging](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level), recreate the issue and add the log file again.

**Comment by tatze96** [comment]
@MitchAnce
Can you please [enable debug logging](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level), reproduce the missing files issue, and send us the log file afterward for further investigation? Ensure to tell us some of the missing files names within the timeframe of the logging output so that we can search for them.

If you don't like to see sensitive data to be shared publicly, please send the log file with the ticket reference GH-16186 to support@cyberduck.io, otherwise attach it here.

**Sidenote**: If you want to share the log file here, make sure to attach it through the GitHub web interface, as it wouldn't be visible to us otherwise.
